packages/driver-adapter-utils/src/index.ts

```diff
@@ -1,5 +1,5 @@
 export { bindAdapter } from './binder'
-export { ColumnTypeEnum, JsonNullMarker } from './const'
```

Comment:
I think `JsonNullMarker` can be removed completely

Reply:
huh, I thought I removed this